src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -304,6 +304,12 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             var functionDetails = await WebFunctionsManager.GetFunctionMetadataResponse(listableFunctions, hostOptions, _hostNameProvider);
             result.Add("functions", new JArray(functionDetails.Select(p => JObject.FromObject(p))));
 
+            // Add host.json to the payload
+            if (_environment.IsKubernetesManagedHosting())
```

Comment:
Is there a strong reason to limit this to Kubernetes? I'd like to also have this change for Antares too.

Reply:
Hi @cgillum 
This is mainly time pressure. Since we need to finish today with testing, 
Scoping it that is not effect existing scenario. Also save time for fixing E2E fails. 
We can create a issue to tracking make this global.

Reply:
We need to scop the changes for Kubernetes Env as we will be taking commit as hotfix and skip OGFs

@TsuyoshiUshio  - Please open a separate issue to remove the check for Kubernetes env after we cut the hotfix release - to ensure change go through full functions release validation

Reply:
Makes sense. Thanks!